app/repositories/user.js

```diff
@@ -83,12 +98,22 @@ export const auth0Login = (navigation) => {
               'modified',
             );
             getUserInformationFromServer(navigation).then(() => {
```

Comment:
This function is not imported. Running `npm run lint:errors` and fixing all issues can avoid such errors.